src/components/guest/GuestFormSchemas.js

```diff
@@ -0,0 +1,68 @@
+import { GuestPriorityStatus } from "models/constants/GuestPriorityStatus";
+import { GuestStatus } from "models/constants/GuestStatus";
+import * as Yup from "yup";
+import { GuestFormFields } from "./GuestFormFields";
+
+const childrenModelSchema = Yup.object().shape({
+    [GuestFormFields.CHILD_AGE]: Yup.number()
+        .integer("common:form.validator.integer")
+        .moreThan(0, "common:form.validator.positiveNumber")
+        .min(1, `common:form.validator.numberMin`),
+});
+
+const phoneNumberRegexp = /^(\+\d{1,2} ?)?\d{3}[- \.]?\d{3}[- \.]?\d{3}$/;
+
+const commonSchema = Yup.object().shape({
+    [GuestFormFields.ADULT_FEMALE_COUNT]: Yup.number()
+        .integer("common:form.validator.integer")
+        .moreThan(0, "common:form.validator.positiveNumber")
+        .min(1, `common:form.validator.numberMin`),
+    [GuestFormFields.ADULT_MALE_COUNT]: Yup.number()
+        .integer("common:form.validator.integer")
+        .moreThan(0, "common:form.validator.positiveNumber")
+        .min(1, "common:form.validator.numberMin"),
```

Comment:
I would say more than -1 and and min 0, right? The group of people can consist of 0 males, 1 female, 0 children. Or: 2 males. Then only condition is that total sum cannot be 0 - use Ref